set socket time out longer

https://github.com/embulk/embulk-input-s3/blob/master/embulk-input-s3/src/main/java/org/embulk/input/s3/AbstractS3FileInputPlugin.java#L218
8 * 60 * 1000

https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/constant-values.html#com.amazonaws.ClientConfiguration.DEFAULT_SOCKET_TIMEOUT
50000


```
Caused by: java.net.SocketTimeoutException: Read timed out
    at java.net.SocketInputStream.socketRead0(Native Method)
    at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
    at java.net.SocketInputStream.read(SocketInputStream.java:171)
    at java.net.SocketInputStream.read(SocketInputStream.java:141)
    at sun.security.ssl.InputRecord.readFully(InputRecord.java:465)
    at sun.security.ssl.InputRecord.readV3Record(InputRecord.java:593)
    at sun.security.ssl.InputRecord.read(InputRecord.java:532)
    at sun.security.ssl.SSLSocketImpl.readRecord(SSLSocketImpl.java:975)
    at sun.security.ssl.SSLSocketImpl.readDataRecord(SSLSocketImpl.java:933)
    at sun.security.ssl.AppInputStream.read(AppInputStream.java:105)
    at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
    at org.apache.http.impl.io.SessionInputBufferImpl.read(SessionInputBufferImpl.java:197)
    at org.apache.http.impl.io.ContentLengthInputStream.read(ContentLengthInputStream.java:176)
    at org.apache.http.conn.EofSensorInputStream.read(EofSensorInputStream.java:135)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at com.amazonaws.event.ProgressInputStream.read(ProgressInputStream.java:180)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at com.amazonaws.event.ProgressInputStream.read(ProgressInputStream.java:180)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at com.amazonaws.util.LengthCheckInputStream.read(LengthCheckInputStream.java:107)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at com.amazonaws.services.s3.internal.S3AbortableInputStream.read(S3AbortableInputStream.java:125)
    at com.amazonaws.internal.SdkFilterInputStream.read(SdkFilterInputStream.java:90)
    at org.embulk.spi.util.InputStreamFileInput.poll(InputStreamFileInput.java:151)
    ... 23 more
```

--- a/src/main/java/org/embulk/input/aurora_mysql_extract_files/AuroraMySQLExtractFilesFileInputPlugin.java
+++ b/src/main/java/org/embulk/input/aurora_mysql_extract_files/AuroraMySQLExtractFilesFileInputPlugin.java
@@ -8,6 +8,7 @@ import java.util.List;
 import java.util.Properties;
 import java.util.stream.Collectors;
 
+import com.amazonaws.ClientConfiguration;
 import com.amazonaws.services.s3.model.DeleteObjectsRequest;
 
 import org.embulk.config.ConfigDiff;
@@ -167,10 +168,24 @@ public class AuroraMySQLExtractFilesFileInputPlugin implements FileInputPlugin {
     private static AmazonS3 newS3Client(PluginTask task) {
         BasicAWSCredentials awsCreds = new BasicAWSCredentials(task.getAwsAccessKey().orNull(),
                 task.getAwsSecretAccessKey().orNull());
+        ClientConfiguration clientConfiguration = getClientConfiguration(task);
         return AmazonS3ClientBuilder
             .standard()
             .withCredentials(new AWSStaticCredentialsProvider(awsCreds))
-            .withRegion(Regions.AP_NORTHEAST_1).build();
+            .withClientConfiguration(clientConfiguration)
+            .withRegion(Regions.AP_NORTHEAST_1)
+            .build();
+    }
+
+    private static ClientConfiguration getClientConfiguration(PluginTask task)
+    {
+        ClientConfiguration clientConfig = new ClientConfiguration();
+
+        clientConfig.setMaxConnections(50); // SDK default: 50
+        clientConfig.setMaxErrorRetry(3); // SDK default: 3
+        clientConfig.setSocketTimeout(8 * 60 * 1000); // SDK default: 50*1000
+
+        return clientConfig;
     }
 
     private List<String> getS3Keys(PluginTask task){

